packages/pn-pa-webapp/src/pages/components/Notifications/ConfirmCancellationDialog.tsx

```diff
@@ -0,0 +1,85 @@
+import {
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogActions,
+  Button,
+  Checkbox,
+  FormControl,
+  FormControlLabel,
+  Typography,
+} from '@mui/material';
+import { t } from 'i18next';
+import React, { useEffect, useState } from 'react';
+
+type Props = {
+  showModal: boolean;
+  onClose: React.MouseEventHandler<HTMLButtonElement>;
+  onConfirm: React.MouseEventHandler<HTMLButtonElement>;
+  payment: boolean;
+};
+
+export default function ConfirmCancellationDialog({
+  showModal,
+  onClose,
+  onConfirm,
+  payment,
+}: Props) {
+  const [checked, setChecked] = useState(false);
+  const handleChange = () => {
+    setChecked(!checked);
+  };
+
+  useEffect(() => {
+    setChecked(false);
+  }, [onClose]);
+
+  return (
+    <Dialog open={showModal} data-testid="modalId">
```

Comment:
Try to use the PnDialog. To understand how to use it, check around the app

Reply:
do